sources/karmashapes_remarks.md

```diff
@@ -0,0 +1,28 @@
+### Karmashapes
+
+Properties with the `ks` prefix are from Karmashapes.
+
+**Examples:** `ks:id` for Karmashapes identifier concordance and `ks:name` for place names. 
+
+Karmashapes is included in Who's On First's administrative data repo in India. Who's on First doesn't always use the Karmashapes geometry, so review that in `src:geom`. When we do, some Karmashapes features source from friendly CC0 or Public Domain licensed projects, and others from a stricter CC-BY project.
+
+Please note the following with regard to Quattroshapes features that require attribution:
+
+* National Geospatial Intelligence Agency (NGA)
+	* URL: https://geonames.nga.mil/geonames/GNSHome/welcome.html
+	* License: Public Domain
+* Socioeconomic Data and Applications Center (SECAC) / Center for International Earth Science Information Network (CIESIN);
+	* URL: https://sedac.ciesin.columbia.edu/data/set/india-india-village-level-geospatial-socio-econ-1991-2001/data-download
+	* License: CC-BY
+* Pradhan Mantri Gram Sadak Yojana (PMGSY);
```

Comment:
Change source name to `Ministry of Rural Development, 2022. PMGSY`

Then the dataset name (layer) is `Pradhan Mantri Gram Sadak Yojana (PMGSY)` as new sub-bullet. That  roughly translates to "Prime Ministers Rural Road Schema" to connect more villages with all weather roads.

Reply:
And no `;` at end of line

Reply:
Above, I see the data layer is called "Habitations".. are these the same data layers?